modules/labs-react/combobox/lib/Combobox.tsx

```diff
@@ -135,7 +135,7 @@ const optionIdPart = `option`;
 export const getOptionId = (baseId?: string, index?: number) =>
   `${baseId}-${optionIdPart}-${index}`;
 
-export const getTextFromElement = (children?: React.ReactNode) => {
+export const getTextFromElement = (children?: React.ReactNode | {}) => {
```

Comment:
it was a temporary solution for fixing type issue with `child === {}` from line 141. But it will be removed because of fully removal of `child === {}`. This check doesn't have sense as it always false (even if child is empty object: `{} === {} // return false`.

Reply:
gotcha thanks for explaining that!